.github/workflows/docker.yml

```diff
@@ -35,6 +42,15 @@ jobs:
           password: "${{ secrets.DockerPassword }}"
           build_extra_args: "--build-arg=ENVIRONMENT=deployment"
 
+      - name: Build and Push unstable Docker image tag
+        uses: whoan/docker-build-with-cache-action@v4
+        with:
+          image_name: ${{ env.IMAGE_NAME }}
+          username: gadockersvc
+          image_tag: ${{ env.UNSTABLE_TAG }}
```

Comment:
We want to push `latest` here too

Reply:
Latest is the default. Pushed in another step. Not sure if the action supports multiple tags. Looking at it in a bit.

Reply:
```
image_tag: Tag(s) of the image to build. Allows multiple comma-separated tags (e.g. one,another) (default: latest).
```
Squashing into one step.